internal/generator/vector/output/security/security.go

```diff
@@ -60,24 +60,27 @@ func NewTLSConf(o logging.OutputSpec, op generator.Options) TLSConf {
 }
 
 func addTLSSettings(o logging.OutputSpec, secret *corev1.Secret, conf *TLSConf) (TLSConf, bool) {
```

Comment:
One of those TLSConf's is too many. The signature of addTLSSettings should be
`(o logging.OutputSpec, secret *corev1.Secret, conf *TLSConf)  bool`
or
`(o logging.OutputSpec, secret *corev1.Secret) *TLSConf`